app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Base class for vaccine errors."""
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    """Raised when visitor has no vaccine."""
+    pass

```

Comment:
Consider adding a descriptive error message to the `NotVaccinatedError` exception. For example: `class NotVaccinatedError(VaccineError): def __init__(self, message='The person is not vaccinated'): super().__init__(message)`.